### PR TITLE
[Magic Firewall] PCAP limits

### PR DESCRIPTION
**Pull request by dcpena**
Added section to call out limits for PCAPs to address PCX-4807.

- Unrelated to ticket but also swapped out dropdown menus for API & Dashboard procedures for tabs.
- Added in formatting to code examples.

### CODE REVIEW

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 1]
```diff
@@ -242,11 +248,18 @@ The capture status displays one of the following options:
...
-  API
-</summary>
- <div class="special-class" markdown="1">
+{{<tabs labels="Dashboard | API">}}
+{{<tab label="dashboard" no-code="true">}}
+
+1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/login) and select **Magic Transit**.
+2. On the **Magic Transit** page next to **Packet captures**, select **Start a capture**.
+3. From the **Packet captures** page, select the **Captures** tab.
+4. Locate your packet capture you want to download and under **Captures**, select the link to open download it.
```
Comment (marciocloudflare) [inline content/magic-firewall/how-to/collect-pcaps.md:257]: Does the link `opens` or `downloads` the capture? Please choose one :)

Comment (marciocloudflare) [inline content/magic-firewall/how-to/collect-pcaps.md]:
```suggestion
4. Locate your packet capture you want to download and under **Captures**, select the link to download it.
```

Comment (marciocloudflare) [reply content/magic-firewall/how-to/collect-pcaps.md]: Based on what the line below says, I've chosen `download` here

File: content/magic-firewall/how-to/collect-pcaps.md [hunk 2]
```diff
@@ -266,32 +279,23 @@ Once the simple PCAP collection is complete, you can download the PCAP by specif
...
 3. From the **Packet captures** page, select the **Captures** tab.
-4. Locate your packet capture you want to download and under **Captures**, select the link to open download it.
-
-Packet captures are available to download when the **Status** displays **Success**. 
-
-</div>
-</details>
 
-## List packet captures
+The list of captures associated with your account displays.
```
Comment (marciocloudflare) [inline content/magic-firewall/how-to/collect-pcaps.md]: I'm not sure if I understand what this final sentence means in this context. It seems there's something missing

File: content/magic-firewall/how-to/form-expressions.md [hunk 3]
```diff
@@ -26,11 +26,15 @@ You can use Magic Firewall to skip or block packets based on source or destinati
...
-    ip.src in { 192.0.2.0/24 }  # good
+```http
+ip.src == 192.0.2.0/24  # bad
+ip.src in { 192.0.2.0/24 }  # good
+```
 
 Expressions have a complexity limit that is easily reached when many joined or nested clauses are in the expression. Here's an example:
 
-    (tcp.dstport == 1000 || tcp.dstport == 1001) && (tcp.dstport == 1002 || tcp.dstport == 1003) && (tcp.dstport == 1004 || tcp.dstport == 1005) && (tcp.dstport == 1006 || tcp.dstport == 1007) && (tcp.dstport == 1008 || tcp.dstport == 1009) && (tcp.dstport == 1010 || tcp.dstport == 1011) && (tcp.dstport == 1012 || tcp.dstport == 1013) && (tcp.dstport == 1014 || tcp.dstport == 1015) && (tcp.dstport == 1016 || tcp.dstport == 1017)
+```http
```
Comment (marciocloudflare) [inline content/magic-firewall/how-to/form-expressions.md]: This may be better as `txt` as it's not HTML code.

**Review by marciocloudflare — APPROVED** [review]

File: content/magic-firewall/how-to/form-expressions.md [hunk 4]
```diff
@@ -26,11 +26,15 @@ You can use Magic Firewall to skip or block packets based on source or destinati
 
 Wirefilter comparisons support CIDR notation, but only inside sets. For example:
 
-    ip.src == 192.0.2.0/24  # bad
-    ip.src in { 192.0.2.0/24 }  # good
+```http
```
Comment (pedrosousa) [inline content/magic-firewall/how-to/form-expressions.md]: Maybe `txt` here.

File: content/magic-firewall/how-to/form-expressions.md [hunk 5]
```diff
@@ -26,11 +26,15 @@ You can use Magic Firewall to skip or block packets based on source or destinati
...
 
-    (tcp.dstport == 1000 || tcp.dstport == 1001) && (tcp.dstport == 1002 || tcp.dstport == 1003) && (tcp.dstport == 1004 || tcp.dstport == 1005) && (tcp.dstport == 1006 || tcp.dstport == 1007) && (tcp.dstport == 1008 || tcp.dstport == 1009) && (tcp.dstport == 1010 || tcp.dstport == 1011) && (tcp.dstport == 1012 || tcp.dstport == 1013) && (tcp.dstport == 1014 || tcp.dstport == 1015) && (tcp.dstport == 1016 || tcp.dstport == 1017)
+```http
+(tcp.dstport == 1000 || tcp.dstport == 1001) && (tcp.dstport == 1002 || tcp.dstport == 1003) && (tcp.dstport == 1004 || tcp.dstport == 1005) && (tcp.dstport == 1006 || tcp.dstport == 1007) && (tcp.dstport == 1008 || tcp.dstport == 1009) && (tcp.dstport == 1010 || tcp.dstport == 1011) && (tcp.dstport == 1012 || tcp.dstport == 1013) && (tcp.dstport == 1014 || tcp.dstport == 1015) && (tcp.dstport == 1016 || tcp.dstport == 1017)
+```
 
 If the limit is reached, the response will have a 400 status code and an error message of "ruleset exceeds complexity constraints." Split the expression into multiple rules and try again.
```
Comment (pedrosousa) [inline content/magic-firewall/how-to/form-expressions.md]:
```suggestion
If the limit is reached, the response will have a `400` status code and an error message of "ruleset exceeds complexity constraints." Split the expression into multiple rules and try again.
```

File: content/magic-firewall/how-to/use-rules-list.md [hunk 6]
```diff
@@ -12,6 +12,7 @@ meta:
 
 In order to use this feature first [create a new list](https://api.cloudflare.com/#rules-lists-create-list).
 
+```bash
```
Comment (pedrosousa) [inline content/magic-firewall/how-to/use-rules-list.md:15]: You can now remove the indentation in the entire code block.

File: content/magic-firewall/how-to/collect-pcaps.md (hunk 1 above)
Comment (dcpena) [reply content/magic-firewall/how-to/collect-pcaps.md]: Good catch, thank you!

File: content/magic-firewall/how-to/form-expressions.md (hunk 3 above)
Comment (dcpena) [reply content/magic-firewall/how-to/form-expressions.md]: Somehow that did not cross my mind, lol. thank you! Fixed

File: content/magic-firewall/how-to/form-expressions.md (hunk 4 above)
Comment (dcpena) [reply content/magic-firewall/how-to/form-expressions.md]: Thank you for this!

File: content/magic-firewall/how-to/form-expressions.md (hunk 5 above)
Comment (dcpena) [reply content/magic-firewall/how-to/form-expressions.md]: Fixed

File: content/magic-firewall/how-to/use-rules-list.md (hunk 6 above)
Comment (dcpena) [reply content/magic-firewall/how-to/use-rules-list.md:15]: Fixed

File: content/magic-firewall/how-to/collect-pcaps.md (hunk 2 above)
Comment (dcpena) [reply content/magic-firewall/how-to/collect-pcaps.md]: I changed this to say "The list of packet captures associated with your account displays." This sentence is here just to let users know what happens after they complete the last step.

**Pull request merged**